test/integration/helpers/forms.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const timing = require('./timing');
+const setInputValue = require('./inputs').setInputValue;
+
+exports.submitNewProjectForm = function(project) {
+  for (let key in project) {
```

Comment:
I'm not crazy about assuming the `project` object keys will match the model keys. I'd rather just fill in the 3 fields we know about.

Reply:
Surprisingly we have this form in a few places now & it's always copied the same way, but I'll update this.

Reply:
is uglier, but still works :)